### PR TITLE
feat(eighth): add bbcu import script

### PR DESCRIPTION
**Pull request by Laur04**
## Proposed changes
- Add script behind `is_eighth_admin` (for now) to allow teachers to mass import their students in eighth period.
- For testing, you need to add the following to `secret.py`.
```
BBCU_SCRIPT = """
// DISCLAIMER: This script is provided as is, with no guarantees.<br>
// To add all students, go to https://us.bbcollab.com/collab/ui/scheduler/session, click on the session,<br>
// click on "invitations", then click "invite attendees". Open the console with Ctrl+Shift+J and paste the following.<br><br>

all = +=+=;
var userNumber = 0;
addUser()
function addUser() {
    $('#add-enrollment-button').click()
    $('#attendee-display-name').val(all[userNumber][0])
    $('#attendee-email').val(all[userNumber][1])
    $('#attendee-display-name').triggerHandler('change')
    $('#attendee-email').triggerHandler('input')
    window.setTimeout( function(){
        $(".button.save-button").click();
        console.log(userNumber);
        userNumber += 1;
        if (userNumber>all.length-1) {
            $(".button.secondary.cancel-button").click();
            $('#panel-return-button').click();
        } else {
            window.setTimeout(addUser, 1000);
        }
    }, 1000)
}
"""
```

### CODE REVIEW

File: intranet/templates/eighth/take_attendance.html [hunk 1]
```diff
@@ -461,6 +475,9 @@ <h3>Passes</h3>
                 <a class="button print-hide" href="{% url 'eighth_email_students' scheduled_activity.id %}">Email Students</a>
                 {% endif %}
                 {% if user.is_eighth_admin %}
+                    <a class="button print-hide" href="#" onclick="openBbcuTab()">View in BbCU Format</a>
+                    <input type="hidden" id="bbcu_script" value="{{ bbcu_script }}">
```
Comment (theo-o) [inline intranet/templates/eighth/take_attendance.html:479]: Minor nit
```suggestion
                        <input type="hidden" id="bbcu_script" value="{{ bbcu_script }}">
```

**Review by etnguyen03 — APPROVED** [review]
LGTM, but the script (to put in `secret.py`) should have semicolons ending each line (Chrome complains about that (`unexpected token function`)).

File: intranet/templates/eighth/take_attendance.html [hunk 2]
```diff
@@ -51,6 +51,21 @@
     });
     </script>
     {% endif %}
+    <script>
+        function openBbcuTab() {
+            var student_list = [];
+            var mem_list = document.getElementsByClassName("bbcu-selector");
```
Comment (theo-o) [inline intranet/templates/eighth/take_attendance.html:57]: You can use jQuery to replace some of these native JS functions

**Review by theo-o — COMMENTED** [review]
`BBCU_SCRIPT` should be defined (perhaps as `None`) in `intranet/settings/__init__.py`

File: intranet/templates/eighth/take_attendance.html (hunk 1 above)
Comment (Laur04) [reply intranet/templates/eighth/take_attendance.html:479]: Why should this be indented beyond the level that the `<a>` tag above it is at?

Comment (theo-o) [reply intranet/templates/eighth/take_attendance.html:479]: Disregard this. This was a skim-based rushed review/

File: intranet/templates/eighth/take_attendance.html (hunk 2 above)
Comment (Laur04) [reply intranet/templates/eighth/take_attendance.html]: Haven't written much in jQuery, is the updated syntax for it about right?  (I did test it and it works)

**Review by theo-o — APPROVED** [review]
LGTM

**Pull request merged**